Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -17,6 +17,8 @@ import Tentacle
 import XCDBLD
 
 class XcodeSpec: QuickSpec {
+	static let currentSwiftVersion = "3.0.2"
```

Comment:
@scottrhoyt time to un-hardcode this — ❌s for https://github.com/Carthage/Carthage/pull/1866#issuecomment-290825024